src/spyglass/common/common_lab.py

```diff
@@ -76,6 +85,47 @@ def insert_from_name(cls, full_name):
             skip_duplicates=True,
         )
 
+    def _load_admin(cls):
+        """Load admin list."""
+        cls.LabMemberInfo._admin = (cls.LabMemberInfo & {"admin": True}).fetch(
+            "datajoint_user_name"
+        )
+
+    @property
+    def admin(cls) -> list:
+        """Return the list of admin users.
+
+        Note: This is cached. If adding a new admin, run _load_admin
+        """
+        if not cls.LabMemberInfo._admin:
```

Comment:
Once _admin has been loaded, this causes an error:
`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`

This is because _admin is an array not a list